.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - LOGS_DIR=/tmp/angular-build/logs
     - BROWSER_PROVIDER_READY_FILE=/tmp/browsersprovider-tunnel-ready
 
+matrix:
+  allow_failures:
+    - env: "JOB=unit BROWSER_PROVIDER=browserstack"
```

Comment:
we could do this for the E2E test failures too, though they haven't been having the same issues